packages/askui-nodejs/src/lib/ui-controller-facade.ts

```diff
@@ -74,6 +75,13 @@ export abstract class UiControllerFacade {
     /* Executable out of the box */
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  protected preStartChecks(): Promise<void> {
+    return new Promise((resolve) => {
+      resolve();
+    });
+  }
```

Comment:
What is this for a trick? 

The promise resolves directly. We don't need it.

Reply:
Ahh I see.

Reply:
Like for the use of the pattern :)